### PR TITLE
Pass correct ResourcePool to VCH for container VMs

### LINKED ISSUE #539: Create container VM in resource pool
Now the installer will create resource pool for VCH, but the container VM is still created out of that resource pool

### PR DESCRIPTION
**Pull request by gigawhitlocks**
Fixes #539

### CODE REVIEW

File: install/management/appliance.go [hunk 1]
```diff
@@ -171,7 +171,8 @@ func (d *Dispatcher) createApplianceSpec(conf *configuration.Configuration) (*ty
 			&types.OptionValue{Key: "guestinfo.vch/sbin/imagec", Value: "-debug -logfile=/var/log/vic/imagec.log -insecure"},
 			&types.OptionValue{Key: "guestinfo.vch/sbin/port-layer-server",
 				Value: fmt.Sprintf("--host=localhost --port=8080 --insecure --sdk=%s --datacenter=%s --cluster=%s --pool=%s --datastore=%s --network=%s --vch=%s",
-					conf.TargetPath, conf.DatacenterName, conf.ClusterPath, conf.ResourcePoolPath, conf.ImageStorePath, conf.ExtenalNetworkPath, conf.DisplayName)},
+					conf.TargetPath, conf.DatacenterName, conf.ClusterPath, conf.ResourcePoolPath+`/`+conf.DisplayName,
```
Comment (emlin) [inline install/management/appliance.go]: Nice job! 
For this change, I would suggest to leverage the fmt.Sprintf previously.
from "--pool=%s" to "--pool=%s/%s", and just put conf.ResourcePoolPath and conf.DisplayName separately.

Comment (gigawhitlocks) [inline install/management/appliance.go]: Oh yes, this is better. My Pythonic tendencies were showing. I'll make this change momentarily

**Comment by emlin** [comment]
LGTM

File: install/management/appliance.go [hunk 2]
```diff
@@ -170,8 +170,9 @@ func (d *Dispatcher) createApplianceSpec(conf *configuration.Configuration) (*ty
 			&types.OptionValue{Key: "guestinfo.vch/components", Value: "/sbin/docker-engine-server /sbin/port-layer-server /sbin/vicadmin"},
 			&types.OptionValue{Key: "guestinfo.vch/sbin/imagec", Value: "-debug -logfile=/var/log/vic/imagec.log -insecure"},
 			&types.OptionValue{Key: "guestinfo.vch/sbin/port-layer-server",
-				Value: fmt.Sprintf("--host=localhost --port=8080 --insecure --sdk=%s --datacenter=%s --cluster=%s --pool=%s --datastore=%s --network=%s --vch=%s",
-					conf.TargetPath, conf.DatacenterName, conf.ClusterPath, conf.ResourcePoolPath, conf.ImageStorePath, conf.ExtenalNetworkPath, conf.DisplayName)},
+				Value: fmt.Sprintf("--host=localhost --port=8080 --insecure --sdk=%s --datacenter=%s --cluster=%s --pool=%s/%s --datastore=%s --network=%s --vch=%s",
+					conf.TargetPath, conf.DatacenterName, conf.ClusterPath, conf.ResourcePoolPath, conf.DisplayName,
```
Comment (hickeng) [inline install/management/appliance.go]: I don't recommend it here as it's potentially fragile, but be aware there is an explicit index mechanism for fmt.
From the docs:
`fmt.Sprintf("%[2]d %[1]d\n", 11, 22)```
will yield "22 11"

Comment (gigawhitlocks) [inline install/management/appliance.go]: Good to know :+1:

**Pull request merged**